networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -774,6 +777,138 @@ TEST_F(CentralityGTest, testApproxClosenessCentralityOnToyGraph) {
     EXPECT_NEAR(0.2, maximum2, tol);
 }
 
+TEST_F(CentralityGTest, testApproxClosenessCentralityOnDirectedToyGraph) {
+    /* Graph:
+     0 -> 3
+     ^   /
+      \ /
+       2    5
+      /  \ /
+     1    4
+    */
+    Graph G(6, false, true);
+
+    G.addEdge(0, 3);
+    G.addEdge(2, 0);
+    G.addEdge(1, 2);
+    G.addEdge(2, 1);
+    G.addEdge(3, 2);
+    G.addEdge(2, 3);
+    G.addEdge(4, 2);
+    G.addEdge(2, 4);
+    G.addEdge(4, 5);
+    G.addEdge(5, 4);
+
+    ApproxCloseness acc(G, 6, 0.1, true);
+    acc.run();
+    std::vector<double> cc = acc.scores();
+    double tol = 0.01;
+
+    EXPECT_NEAR(acc.maximum(), 0.2, tol);
+    EXPECT_NEAR(0.38, cc[0], tol);
```

Comment:
Avoid Yoda-style conditions https://en.wikipedia.org/wiki/Yoda_conditions, also below.
```suggestion
    EXPECT_NEAR(cc[0], 0.38, tol);
```

Reply:
You can also define a vector of expected values and do the checks in a for loop.